doc/source/buildoptions.rst

```diff
@@ -33,25 +33,20 @@ e.g. ``--requirements=python3``.
 
 
 CrystaX python3
-###############
-
-.. warning:: python-for-android originally supported Python 3 using the CrystaX
-             NDK. This support is now being phased out as CrystaX is no longer
-             actively developed.
-
-.. note:: You must manually download the `CrystaX NDK
-   <https://www.crystax.net/android/ndk>`__ and tell
-   python-for-android to use it with ``--ndk-dir /path/to/NDK``.
-
-Select this by adding the ``python3crystax`` recipe to your
-requirements, e.g. ``--requirements=python3crystax``.
-
-This uses the prebuilt Python from the `CrystaX NDK
-<https://www.crystax.net/android/ndk>`__, a drop-in replacement for
-Google's official NDK which includes many improvements. You
-*must* use the CrystaX NDK 10.3.0 or higher when building with
-python3. You can get it `here
-<https://www.crystax.net/en/download>`__.
+~~~~~~~~~~~~~~~
+
+python-for-android originally supported Python 3 using the CrystaX NDK.  Since
+we have a working python3 recipe, we don't support CrystaX NDK anymore. If you
```

Comment:
I would be more direct with this kind of doc, you can just write 

> python-for-android no longer supports building for Python 3 using the CrystaX NDK. Instead, use the `python3` recipe, which can be built using the normal Google NDK.

> The last python-for-android version supporting CrystaX was 0.7.0.

No need for this to hold up merging the PR, we can improve the doc later (there are plenty of parts that I wrote that also need improvement), just noting as something I've been thinking about.